docs/umlDiagrams/HelpSequence.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+autonumber
+
+-> HelpCommand: execute()
```

Comment:
We can add the ":" to front of class by typing ":Class".

Reply:
will have some error tho

Reply:
okay nvm updated